### PR TITLE
Remove follow games button from onboarding follows step

### PR DESCRIPTION
**Pull request by YLivay**

### CODE REVIEW

File: src/app/components/forms/onboarding/follows/follows.vue [hunk 1]
```diff
@@ -4,40 +4,20 @@
...
 					</translate>
 				</h3>
 
 				<p class="text-muted">
-					<translate>
-						Get connected right away with some of the best games on Game Jolt.
-					</translate>
+					<span v-translate>
+						We have fanart, lets plays, guides,
+						<i>DANK</i>
```
Comment (nilllzz) [inline src/app/components/forms/onboarding/follows/follows.vue]: Need corporate approval here.

Comment (hworld) [reply src/app/components/forms/onboarding/follows/follows.vue]: "Explore fan-created artwork, videos, game guides and more."

**Comment by hworld** [comment]
Looks good once you update the copy

**Comment by hworld** [comment]
You should remove the backend code too so that we don't keep code around that we're not using.

File: src/app/components/forms/onboarding/follows/follows.vue (hunk 1 above)
Comment (YLivay) [reply src/app/components/forms/onboarding/follows/follows.vue]: aww

**Pull request merged**